content/en/synthetics/api_tests/udp_tests.md

```diff
@@ -66,7 +66,7 @@ You can set alert conditions to determine the circumstances under which a test s
 
 #### Fast retry
 
-Your test can trigger retries in case of failed test result. By default, the retries are performed 300 ms after the first failed test result. The retry interval can be configured with the [API][5].
+Your test can trigger retries in case of failed test result. Configuration is available to better suit your alerting sensibility.
```

Comment:
```suggestion
Your test can trigger retries `X` times after `Y` ms in case of a failed test result. Customize the retry interval to suit your alerting sensibility.
```